docs/api/client/hoodie.account.rst

```diff
@@ -9,10 +9,726 @@ user </camp/hoodieverse/glossary.html#private-user-store>`__, it makes
+        alert('Hello there, ' + accountProperties.username)
+    })
+
+account.off
+-----------
+
+Removes event handler that has been added before
+
+.. code:: js
+
+    account.off(event, handler)
+
+Example
+
+.. code:: js
+
+    account.off('singin', showNotification)
+
+Events
+------
+
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| Event              | Description                                                                     | Arguments                                        |
++====================+=================================================================================+==================================================+
+| ``signup``         | New user account created successfully                                           | ``accountProperties`` with ``.session property`` |
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``signin``         | Successfully signed in to an account                                            | ``accountProperties`` with ``.session property`` |
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``signout``        | Successfully signed out                                                         | ``accountProperties`` with ``.session property`` |
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``passwordreset``  | Email with password reset token sent                                            |                                                  |	
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``unauthenticate`` | Server responded with "unauthenticated" when checking session                   |                                                  |	
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``reauthenticate`` | Successfully signed in with the same username (useful when session has expired) | ``accountProperties`` with ``.session property`` |
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+| ``update``         | Successfully updated an account's properties                                    | ``accountProperties`` with ``.session property`` |
++--------------------+---------------------------------------------------------------------------------+--------------------------------------------------+
+
+Hooks
```

Comment:
I did the changes @gr2m now we can merge it i think :)
@pmbanugo By the way yes me should definitely add the things you mentioned in anither issue